### ISSUE TITLE
[bug] error in GUI 

### ISSUE THREAD

**Issue by jpdeleon** [MNGuenther/allesfitter#11]
Thank you for this amazing package. I ran allesitter to do transit fitting in the past and it worked fine. Now I ran with phase curve fitting using the most updated version in github but I encountered a bug i.e. "missing key ln_yerr_median" in part 6.6 "tighter priors on errors and baseline" in GUI that does not allow me proceed to the last part. Do you need more details to debug this problem?

![image](https://user-images.githubusercontent.com/15188345/89279748-b2147880-d682-11ea-8691-8694d977c4c2.png)

**Comment by MNGuenther** [comment]
Hi @jpdeleon, 
Thanks for raising this; it should be a quick and minor fix. 
A few versions ago I had renamed "log" into "ln" to make it clear that it is a natural logarithm (and not log10, for example). But I forgot to change that in the file headers, so if you check your "priors" folder, you will see they are still "log_yerr_median" etc. 
Can you replace your local files called "estimate_noise.py" and ""estimate_noise_wrap.py" with the attached ones and let me know if this works now?
[attachment.zip](https://github.com/MNGuenther/allesfitter/files/5074120/attachment.zip)

**Comment by jpdeleon** [comment]
Hi @MNGuenther, thanks for providing the solution which fixed my problem. However, there is another minor issue which shows up during sampling:
```
You are using outdated keywords. Automatically renaming baseline_gp1_flux_[name] ---> baseline_gp_matern32_lnsigma_flux_[name]. Please fix this before the Duolingo owl comes to get you.
```

ps. Do you need a pull request from me after I incorporate your proposed fixes?

**Comment by MNGuenther** [comment]
Great! 
No need to worry about the warning you see for the outdated keywords, you can just ignore that for now, and we will update it in the next release.
We will push all these fixes with version 1.1.2 in the next weeks, so no need for any pull request.
